shared/teams/team/rows/index.tsx

```diff
@@ -126,10 +146,13 @@ export const useChannelsSections = (teamID: Types.TeamID, shouldActuallyLoad: bo
       key: 'channel-channels',
       renderItem: ({item}) => <ChannelRow teamID={teamID} channel={item} />,
     },
-    {data: ['channel-info'], key: 'channel-info', renderItem: () => <ChannelFooterRow />},
+    channelMetas?.size < 5 && yourOperations.createChannel
```

Comment:
What should happen if `channelMetas` is undefined?

Reply:
I think an info panel instead of a "there are no channels here" feels fine for me for that state. But also we're adding a loading panel (the TODO on line 132).